resources/js/common/components/AchievementsListItem/AchievementsListItem.tsx

```diff
@@ -172,7 +172,11 @@ export const AchievementsListItem: FC<AchievementsListItemProps> = ({
             </p>
 
             <p className="mb-0.5 flex gap-x-1 text-2xs md:mb-0 md:justify-center md:text-center">
-              <ProgressBarMetaText achievement={achievement} playersTotal={playersTotal} />
+              <ProgressBarMetaText
+                achievement={achievement}
+                playersTotal={playersTotal}
+                unlockPercentage={unlockPercentage}
```

Comment:
I see this fixing the issue, but I don't understand why.
```
const unlockPercentage = achievement.unlockPercentage ? Number(achievement.unlockPercentage) : 0;
```

Is it just the changes from `unlockHardcorePercentage` to `unlockPercentage`?
```
const unlockHardcorePercentage = achievement.unlockHardcorePercentage ?? 0;
```
If so, wouldn't that be the easier fix?